client/src/test/java/com/alibaba/nacos/client/logging/log4j2/Log4J2NacosLoggingTest.java

```diff
@@ -33,15 +33,12 @@
     
     @Test
     public void testLoadConfiguration() {
-        LoggerContext loggerContext = (LoggerContext) LogManager.getContext(false);
-        Configuration contextConfiguration = loggerContext.getConfiguration();
-        Assert.assertEquals(0, contextConfiguration.getLoggers().size());
```

Comment:
Why did you remove this logic?

Reply:
> Why did you remove this logic?

because `nacos-client` dependency `log4j` and `logback`, but actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory], and [the  binding is determined by the JVM](http://www.slf4j.org/codes.html#multiple_bindings), so when run `mvn clean install -DskipITs` , the method `NacosLogging.getInstance().loadConfiguration()` is executed in `com.alibaba.nacos.client.utils.LogUtils`, and 
 `contextConfiguration.getLoggers().size()` is more than one.

why it is work well before i modifed this issue? because effective logger is log4j, but determined logback, executed `LogbackNacosLogging().loadConfiguration()`, `contextConfiguration.getLoggers().size()` still zero.

Reply:
But CI can pass, which means the old implementation is correct.

If can't remove this logic can't pass this case after your changes. We need more consider your changes whether has problem.

Reply:
> But CI can pass, which means the old implementation is correct.
> 
> If can't remove this logic can't pass this case after your changes. We need more consider your changes whether has problem.

ok, thanks for your reply,  But I still think just executed `Class.forName("ch.qos.logback.classic.Logger")` whthout exception, determined current logger instance is logback, is not particularly appropriate.